pkg/machine/actuator.go

```diff
@@ -142,12 +143,20 @@ func (oc *OpenstackClient) convertMachineToCapoInstanceSpec(osc *openStackContex
 		return nil, maoMachine.InvalidMachineConfiguration("error creating bootstrap for %s: %v", machine.Name, err)
 	}
 
+	var createAddressPairs bool = true
+	if clusterInfra.Status.PlatformStatus.OpenStack.LoadBalancer != nil && clusterInfra.Status.PlatformStatus.OpenStack.LoadBalancer.Type != "" && clusterInfra.Status.PlatformStatus.OpenStack.LoadBalancer.Type == configv1.LoadBalancerTypeUserManaged {
```

Comment:
`clusterInfra.Status.PlatformStatus.OpenStack.LoadBalancer.Type != ""` is implied when you have `clusterInfra.Status.PlatformStatus.OpenStack.LoadBalancer.Type == configv1.LoadBalancerTypeUserManaged`.
```suggestion
	if clusterInfra.Status.PlatformStatus.OpenStack.LoadBalancer != nil && clusterInfra.Status.PlatformStatus.OpenStack.LoadBalancer.Type == configv1.LoadBalancerTypeUserManaged {
```